backend/src/components/fundingAgreements.js

```diff
@@ -34,8 +36,7 @@ async function getFundingAgreements(req, res) {
     }
     return res.status(HttpStatus.OK).json(fundingAgreements)
   } catch (e) {
-    log.error(e)
-    return res.status(HttpStatus.INTERNAL_SERVER_ERROR).json(e.data ? e.data : e?.status)
+    handleError(res, e)
```

Comment:
Great question. You don't actually need to return res.status unless you want to end processing. Since it's the end of the block it's not required. PS I looked this up :).